rest/javascript/cosmwasm_client/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "pull_service_client",
+  "version": "1.0.0",
+  "description": "I've created a JavaScript library that streamlines gRPC requests to acquire Oracle proofs. These Oracle proofs are then put to use within the main.js file, where an illustrative example showcases how to integrate them. Users have the flexibility to customize the callContract function in the main.js file according to their specific needs.",
```

Comment:
it should be,
`I've created a JavaScript library that streamlines rest/rpc requests to acquire Oracle proofs`